### ISSUE TITLE
Due to the change in the deviceID, device status cannot be reported correctly.

### ISSUE THREAD

**Issue by wbc6080** [kubeedge/kubeedge#5407]
**What happened**:
Currently, mapper cannot correctly report device status messages to the cloud.
**What you expected to happen**:
Correctly report device status messages to the cloud
**How to reproduce it (as minimally and precisely as possible)**:
Build and run the mapper normally, create the device, and specify in the device configuration file to report to the cloud.
**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`): v1.27.7
- KubeEdge version(e.g. `cloudcore --version` and `edgecore --version`): v1.16.0
- <details><summary>Cloud nodes Environment:</summary>

  - Hardware configuration (e.g. `lscpu`):
  - OS (e.g. `cat /etc/os-release`):
  - Kernel (e.g. `uname -a`):
  - Go version (e.g. `go version`):
  - Others:

  </details>
- <details><summary>Edge nodes Environment:</summary>

  - edgecore version (e.g. `edgecore --version`):
  - Hardware configuration (e.g. `lscpu`):
  - OS (e.g. `cat /etc/os-release`):
  - Kernel (e.g. `uname -a`):
  - Go version (e.g. `go version`):
  - Others:

  </details>

**Comment by wbc6080** [comment]
**problem analysis**

This problem occurred after solving #5220 . We believe that it is because we changed the definition of deviceID to the form of device.namespace/device.name, which caused problems in message parsing.
After analysis we found the problem. After mapper sends the twin-update message to edgecore, the `process` file of `devicetwin` will parse various messages. Since the deviceID is in the form of device.namespace/device.name, and each part of the message is also connected by the `/` symbol, the parsing time is one more bit will be parsed than before, resulting in a mismatch.
![消息解析问题](https://github.com/kubeedge/kubeedge/assets/48212997/b122705f-8f03-45ed-a7f8-962d1958bba7)
